core/modules/dashboard/includes/block.create.inc

```diff
@@ -20,66 +20,86 @@ class DashboardCreateBlock extends Block {
-    return t('Content creation links');
+    return t('Content management links');
   }
 
   /**
    * {@inheritdoc}
    */
   function getAdminPreview() {
-    return t('Links to create new content.');
+    return t('Links to administer content types.');
   }
 
   /**
    * {@inheritdoc}
    */
   function getContent() {
     $node_types = node_type_get_types();
-    $no_access = TRUE;
-    $create = array();
+    $access = user_access('access the content overview page');
 
-    foreach ($node_types as $machine => $node_type) {
-      // If enabled or all types are enabled.
-      if (in_array($machine, $this->settings['types']) || empty($this->settings['types'])) {
-        // Check access, then add a link to create content.
-        if (node_access('create', $machine)) {
-          $type_url_str = str_replace('_', '-', $node_type->type);
-          $create[] = l(t('Add new @type', array('@type' => t($node_type->name))), 'node/add/' . $type_url_str, array('html' => TRUE));
-          $no_access = FALSE;
+    $header = array(
+      array('data' => t('Content type')),
+      array('data' => t('Add new')),
+      array('data' => t('Manage')),
+    );
+    $rows = array();
+
+    // Check access, then add a link to create content.
+    if ($access) {
+      foreach ($node_types as $machine => $node_type) {
+        // If no types are specified, show all types.
+        if (empty($this->settings['types']) || in_array($machine, $this->settings['types'])) {
+          $type_url_string = str_replace('_', '-', $node_type->type);
+          $create = l(t('Add new', array('@type' => t($node_type->name))), 'node/add/' . $type_url_string, array('html' => TRUE));
+          $manage = l(t('Manage', array('@type' => t($node_type->name))), 'admin/content/node', array('html' => TRUE, 'query' => array('status' => 'All', 'type' => $type_url_string)));
+          $rows[] = array(
+            'data' => array(
+              check_plain(t($node_type->name)),
+              $create,
+              $manage,
+            ),
+          );
         }
       }
     }
-
-    // If there are existing content types, but user has no access to create any
-    // of them, hide the block completely.
-    if ($no_access) {
+    else {
       return array();
     }
 
-    $build = array(
-      '#theme' => 'dashboard_panel__create',
-    );
-    if (empty($create)) {
-      $build['list'] = array(
-        '#type' => 'markup',
-        '#markup' => t('No content types have been configured yet.'),
+    if (empty($rows)) {
+      $rows[] = array(
+        array(
+          'data' => t('There are no content types to display.'),
+          'colspan' => 4,
+        ),
       );
     }
-    else {
-      $build['list'] = array(
-        '#theme' => 'item_list',
-        '#items' => $create,
+
+    $panel = array(
+      '#theme' => 'dashboard_panel__node_types',
+    );
+    $panel['table'] = array(
+      '#theme' => 'table',
+      '#header' => $header,
+      '#rows' => $rows,
```

Comment:
`theme_table()` supports an `empty` variable. Have you tried something like this?:
```suggestion
      '#rows' => $rows,
      '#empty' =>t('There are no content types to display.'),
```